app/pdf_utils/fill_sections.py

```diff
@@ -73,6 +73,9 @@ def get_paragraph_text(section_element):
         #         # if text_leaf.get(option) and e.strip(" ") != "":
         #         section_p_text = section_p_text.strip(" ") if section_p_text.strip(" ") != "" else section_p_text
 
+        #         # escape latex
```

Comment:
@thenav56 
Hi Navin,
Could you take a look at the latest update to this PR which includes adding, formatting, and generating PDF content.
Very similar to SNWG.

Context:
- I have refactored the PDF generation logic so that each document section is "scanned" for the type of content in it's section, and then I have separated the preparation of PDF media contents into `pdf_utils`. The `pdf_utils` are utilized within `fill_contents` and `fill_sections`. Then these items are appended in order to a list that is part of the global variable `SECTIONS`.
Finally, in `pdf.generator`, the documents' media contents are then appended to the LaTex document.
- There is some older code in `generator.py` that can/will be removed at a later time. 

Situation/Issue:

The issue is that currently inline text formatting is not compiling properly in LaTex.
- The commented lines in attempt to apply `utils.escape_latex` and `command(section_p_text` to the document text strings.
- the `escape_latex` function disrupts LaTex compiling and the document does not compile correctly.

If I leave out `utils.escape_latex` then the document's formatting (italic, bold, etc) shows as regular text in the document and does not actually apply the formatting.

I have also tried applying `NoEscape` to the [text here in](https://github.com/NASA-IMPACT/nasa-apt/blob/0663fe21c592355ceb1010a5d695794b748182a4/app/pdf_utils/fill_contents.py#L38) `pdf_utils.fill_contents`. However, this also causes LaTex compilation errors.

Goal:
- Allow PDF text rich formatting (bold, italic, etc) without breaking LaTex
- Include linebreaks after each paragraph and media content in the PDF.

Reply:
Here is a current example PDF:
[ozone-mapping-and-profiler-suite-v1-0-journal (35).pdf](https://github.com/NASA-IMPACT/nasa-apt/files/10190424/ozone-mapping-and-profiler-suite-v1-0-journal.35.pdf)